fork-common/src/main/java/com/shazam/fork/suite/TestSuiteLoader.java

```diff
@@ -79,7 +88,8 @@ private TestCaseEvent convertToTestCaseEvent(ClassDefItem classDefItem,
         AnnotationItem[] annotations = method.annotationSet.getAnnotations();
         String testClass = getClassName(classDefItem);
         boolean ignored = isClassIgnored(annotationDirectoryItem) || isMethodIgnored(annotations);
-        return newTestCase(testMethod, testClass, ignored);
+        List<String> permissionsToRevoke = revokeAnnotations(annotations);
```

Comment:
The method name suggests that it returns the annotations themselves. Should it be renamed to permissionsFrom(...) or getPermissionsToRevoke(...)?

Reply:
👍